### ISSUE TITLE
Tutorial 2: using IncrementalInference failing

### ISSUE THREAD

**Issue by jim-hill-r** [NavAbility/BinderNotebooks#41]
using IncrementalInference is failing when running Tutorial 2 Caesar version in Binder.

To reproduce: Go to the following link and run the tutorial from the top. Cell 3 is failing unless you uncomment the Pkg imports.

https://mybinder.org/v2/gh/NavAbility/BinderNotebooks/30173aa54f1d34be92ceed50f72601236949b294?urlpath=lab%2Ftree%2Fjulia%2Fcaesar-api%2Ficra-2-nongaussian.ipynb

**Comment by dehann** [comment]
Yeah i added the package add after rubbing into the same issue.  Looks like it's for first runs only.  I'm not sure how to fix, and actually have a related question on this, does anyone know if we can compile and ship with a precompiled image for fast first time to plot into these binders?

**Comment by Affie** [comment]
Notebook link looks outdated, 20 days ago:
![1371C5D7-C6B0-461D-9ACF-DA70351724FA](https://user-images.githubusercontent.com/6612981/160249506-42dfaab2-ab48-4d9e-afff-e57cd1c7ae3a.jpeg)

Latests seems fine to me

**Comment by dehann** [comment]
I'm trying to find somethign on precompiling the image, but not quite finding something yet.  Lots of circumstance though:
- https://discourse.julialang.org/t/packagecompiler-mybinder-template/22756/4 (necessary link doesnt work)
- https://github.com/mitmath/binder-env
- https://github.com/gvarnavi/binder-env_py-3.8_jl-1.6_wl-12.2

**Comment by dehann** [comment]
https://github.com/gvarnavi/binder-env_py-3.8_jl-1.6_wl-12.2/blob/9aa7e11c1c00ebea61811582caf8bb5c5bf8ad57/Dockerfile#L122
